templates/rbi/prism/node.rbi.erb

```diff
@@ -148,7 +148,7 @@ end
 module Prism::<%= flag.name %>
   <%- flag.values.each_with_index do |value, index| -%>
   # <%= value.comment %>
-  <%= value.name %> = T.let(1 << <%= index %>, Integer)
+  <%= value.name %> = T.let(1 << <%= index + 2 %>, Integer)
```

Comment:
I think it'd be good to add `COMMON_FLAGS = 2` in https://github.com/ruby/prism/blob/main/templates/template.rb to avoid hardcoding that value in a bunch of places.

Reply:
Done